packages/react-app/components/notifications/OnboardingBatch.jsx

```diff
@@ -1,38 +1,133 @@
-import { Box, CloseButton, Flex, Text, Button, Link, Image, Spacer, Tooltip } from "@chakra-ui/react";
 import React from "react";
+import {
+  Box,
+  CloseButton,
+  Flex,
+  Text,
+  Button,
+  Link,
+  Image,
+  Spacer,
+  Tooltip,
+  useToast,
+  useColorModeValue,
+} from "@chakra-ui/react";
+import { getUpdateBatchSignMessage, postUpdateBatch } from "../../data/api";
+import { useUserAddress } from "eth-hooks";
 
 const NEXT_BATCH_TELEGRAM = process.env.NEXT_PUBLIC_BATCH_TELEGRAM;
 const NEXT_BATCH_DATE = process.env.NEXT_PUBLIC_BATCH_DATE;
+const NEXT_BATCH = process.env.NEXT_PUBLIC_BATCH_NUMBER;
```

Comment:
I think another good idea would be to have a database collection with all the Batch data, where we store the batch number, telegram link, contract, dates, status (open, etc) etc....

This can help us to make things more dynamic (Better that env vars) What do you think?)

(it can be in a diff PR if you prefer)